pennylane_lightning/src/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -38,10 +38,20 @@ namespace Pennylane::Gates {
  * @tparam PrecisionT Floating point precision of underlying statevector data
  */
 class GateImplementationsLM : public PauliGenerator<GateImplementationsLM> {
+  private:
+    /* Alias utility functions */
+    static constexpr auto fillLeadingOnes = Util::fillLeadingOnes;
```

Comment:
Could this not become `using fillLeadingOnes = Util::fillLeadingOnes;` or even `using namespace Util;` ? Or is there an advantage of doing it this way?

Reply:
Fixed with adding a private function.